src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booksList: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      const newBook = {
+        id: 4,
+        title: 'Book',
+        category: 'cat',
+        author: 'me',
+        completed: 45,
+        chapter: 'chapter 89',
+      };
+      state.booksList = [...state.booksList, newBook];
+    },
```

Comment:
Nice job so far👍. Just as required for this milestone, the addBook reducer should currently return an empty array😉.
<img width="397" alt="image" src="https://github.com/sahlminkok/BookStore/assets/64643995/794c783d-a2b7-4df0-8642-c21a921fa840">
I suggest you remove the books you have added and configure the addBook reducer to return an empty array of books as required👍.

Reply:
@Ibizugbe but the initial state is empty I guess and here I'm just declaring a new state after dispatch, please consider and guide me through how I'm going to make this work.